pkg/common-controller/snapshot_controller.go

```diff
@@ -1374,3 +1391,61 @@ func (ctrl *csiSnapshotCommonController) setAnnVolumeSnapshotBeingDeleted(conten
 	}
 	return nil
 }
+
+// addInvalidContentLabel adds a label to invalid content objects.
+func (ctrl *csiSnapshotCommonController) checkAndAddInvalidContentLabel(content *crdv1.VolumeSnapshotContent) error {
+	if utils.MapContainsKey(content.ObjectMeta.Labels, utils.VolumeSnapshotContentInvalidLabel) {
+		return nil
+	}
+	if err := webhook.IsValidSnapshotContent(content); err != nil {
```

Comment:
IsXXXX by convention should return a boolean value rather than error

Reply:
renamed